container_images/gotest/main.sh

```diff
@@ -36,7 +36,7 @@ fi
 cp /go-test.txt ${ARTIFACTS}/
 
 # $ARTIFACTS is provided by prow decoration containers
-go tool cover -html=/coverage.out -o ${ARTIFACTS}/coverage.html
+go tool cover -func=/coverage.out | grep ^total | awk '{print $NF}' | tr -d '%' > ${ARTIFACTS}/coverage.txt
```

Comment:
Suggestion: save the full output in a place that will let use analyze where coverage is low/high (unless the junit report already does this for us).

Reply:
no, the jUnit here only has test results. and it isn't saved in this invocation either.

you mean you want the 'coverage.out' file to be kept? I think we only ever need the latest, and if we come up with a use for it, we would wire it here. Otherwise I don't think it's necessary to create an archive of coverage details at different historical times.

Reply:
Ah is there a place where we keep the latest info already? I was mostly interested in just understanding what the current coverage is for the latest release (so we can understand if we have a directory that we think should be covered, but isn't).